CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
-# unrelease
+# 0.2.0 - 2024-02-27
 
 - Bump MSRV to 1.56.1
+- Re-write the `FromHex` trait [#80](https://github.com/rust-bitcoin/hex-conservative/pull/80)
+- For improved ergonomics, add a prelude module [#36](https://github.com/rust-bitcoin/hex-conservative/pull/36)
+- Add a `serde` module [#37](https://github.com/rust-bitcoin/hex-conservative/pull/37)
+- Remove arbitrary padding limit [#41](https://github.com/rust-bitcoin/hex-conservative/pull/41)
+- Make `fmt_hex_exact` honour `Formatter::precision`[#81](https://github.com/rust-bitcoin/hex-conservative/pull/81)
+
+## Performance improvements
+
+- Revamp `BufEncoder` [](https://github.com/rust-bitcoin/hex-conservative/pull/52)
```

Comment:
Also changes API, so we might list it under breaking changes. There's currently no "breaking changes" section but I believe we should have it.

Reply:
I added
```
There are a bunch of breaking changes in this release, 

- The `FromHex` trait has changed considerably. 
- The `BufEncoder` got revamped.
- A bunch of public errors have changed.
```